examples/camera/camera_orbit.rs

```diff
@@ -0,0 +1,163 @@
+            material: materials.add(StandardMaterial {
+                base_color: Color::srgb(0.3, 0.5, 0.3),
+                // Turning off culling keeps the plane visible when viewed from beneath.
+                cull_mode: None,
+                ..default()
+            }),
+            ..default()
+        },
+    ));
+
+    commands.spawn((
+        Name::new("Cube"),
+        PbrBundle {
+            mesh: meshes.add(Cuboid::default()),
+            material: materials.add(Color::srgb(0.8, 0.7, 0.6)),
+            transform: Transform::from_xyz(1.5, 0.51, 1.5),
+            ..default()
+        },
+    ));
+
+    commands.spawn((
+        Name::new("Light"),
+        PointLightBundle {
+            transform: Transform::from_xyz(3.0, 8.0, 5.0),
+            ..default()
+        },
+    ));
+}
+
+fn instructions(mut commands: Commands) {
+    commands
+        .spawn((
+            Name::new("Instructions"),
+            NodeBundle {
+                style: Style {
+                    align_items: AlignItems::Start,
+                    flex_direction: FlexDirection::Column,
+                    justify_content: JustifyContent::Start,
+                    width: Val::Percent(100.),
+                    ..default()
+                },
+                ..default()
+            },
+        ))
+        .with_children(|parent| {
+            parent.spawn(TextBundle::from_section(
+                "W or S: pitch",
+                TextStyle::default(),
+            ));
+            parent.spawn(TextBundle::from_section(
+                "A or D: yaw",
+                TextStyle::default(),
+            ));
+            parent.spawn(TextBundle::from_section(
+                "Q or E: roll",
+                TextStyle::default(),
+            ));
+        });
+}
+
+fn orbit(
+    mut camera: Query<&mut Transform, With<Camera>>,
+    camera_settings: Res<CameraSettings>,
+    keyboard_input: Res<ButtonInput<KeyCode>>,
+    time: Res<Time>,
+) {
+    let mut transform = camera.single_mut();
+
+    let mut delta_pitch = 0.0;
+    let mut delta_roll = 0.0;
+    let mut delta_yaw = 0.0;
+
+    if keyboard_input.pressed(KeyCode::KeyW) {
+        delta_pitch += camera_settings.orbit_speed;
+    }
+    if keyboard_input.pressed(KeyCode::KeyS) {
+        delta_pitch -= camera_settings.orbit_speed;
+    }
+
+    if keyboard_input.pressed(KeyCode::KeyQ) {
+        delta_roll -= camera_settings.orbit_speed;
+    }
+    if keyboard_input.pressed(KeyCode::KeyE) {
+        delta_roll += camera_settings.orbit_speed;
+    }
+
+    if keyboard_input.pressed(KeyCode::KeyA) {
+        delta_yaw -= camera_settings.orbit_speed;
+    }
+    if keyboard_input.pressed(KeyCode::KeyD) {
+        delta_yaw += camera_settings.orbit_speed;
+    }
+
+    // Incorporating the delta time between calls prevents this from being framerate-bound.
+    delta_pitch *= time.delta_seconds();
+    delta_roll *= time.delta_seconds();
+    delta_yaw *= time.delta_seconds();
```

Comment:
```suggestion
    let speed = camera_settings.orbit_speed * time.delta_seconds();
    delta_pitch *= speed;
    delta_roll *= speed;
    delta_yaw *= speed;
```
Bit easier to understand where the numbers come from this way :)